azure-pipelines.yml

```diff
@@ -176,71 +175,92 @@ steps:
       GdnBreakGdnToolCredScanSeverity: Warning
       GdnBreakGdnToolESLint: true
       GdnBreakGdnToolESLintSeverity: Warning
-      GdnBreakPolicy: M365
+      GdnBreakPolicy: M365  
+
+  - task: AzureRmWebAppDeployment@4
+    condition: and(
+      in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'),
+      startsWith(variables['Build.SourceBranch'], 'refs/tags/v'))
+    inputs:
+      ConnectionType: 'AzureRM'
+      azureSubscription: '$(AzureSubscription)'
+      appType: 'webApp'
+      WebAppName: 'musicalsink'
+      package: '$(Build.ArtifactStagingDirectory)\teams-test-app\$(Build.BuildId).zip'
+    displayName: 'Deploy to musical sink'
```

Comment:
nit: I think the displayName should reference the webappname, i.e., `'Deploy to musicalsink'`. Same for `'Deploy to cloudroll'`